sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -854,7 +857,8 @@ public <T extends AbstractDomainObject> Predicate createCriteriaFilter(CaseCrite
 					CriteriaBuilderHelper
 						.unaccentedIlike(cb, personQueryContext.getSubqueryExpression(PersonQueryContext.PERSON_PRIMARY_OTHER_SUBQUERY), textFilter),
 					CriteriaBuilderHelper.unaccentedIlike(cb, joins.getPersonAddress().get(Location.CITY), textFilter),
-					CriteriaBuilderHelper.ilike(cb, joins.getPersonAddress().get(Location.POSTAL_CODE), textFilter)));
+					CriteriaBuilderHelper.ilike(cb, joins.getPersonAddress().get(Location.POSTAL_CODE), textFilter),
+					CriteriaBuilderHelper.unaccentedIlike(cb, joins.getPerson().get(Person.NATIONAL_HEALTH_ID), textFilter)));
```

Comment:
I would say `ilike` is good enough as these id's are numbers in general, please change in all other places

Reply:
updated in all services (5 places)